src/net/sourceforge/kolmafia/request/UseItemRequest.java

```diff
@@ -6103,6 +6103,10 @@ else if (responseText.contains("You can't figure out the angle")) {
         }
         return;
 
+      case ItemPool.REPLICA_SOURCE_TERMINAL:
```

Comment:
Since TerminalRequest does this:
```
    if (!KoLCharacter.inNuclearAutumn()
        && !KoLConstants.campground.contains(ItemPool.get(ItemPool.SOURCE_TERMINAL))) {
      KoLmafia.updateDisplay(KoLConstants.MafiaState.ERROR, "You don't have a Source terminal.");
      return;
    }
```
I'd say that the original Source Terminal has the same issue.
Although using that item is almost infinitely more rare than using a replica Source Terminal, these days.

Reply:
very likely!

Also you only use the Source Terminal proper once, while you use the replica every run.